### ISSUE TITLE
Cannot set amplitude on channel 0

### ISSUE THREAD

**Issue by einsmein** [zhinst/zhinst-qcodes#54]
I was trying to set an amplitude on the MFLI, which I assume would be `mfli1.sigouts[0].amplitudes[0].value()`. I got an error that there is no attribute `value`. When I checked the snapshot, there is no parameters on 0th channel. 
```
> mfli1.sigouts[0].amplitudes[0].snapshot()
{'functions': {},
 'submodules': {},
 'parameters': {},
 '__class__': 'zhinst.qcodes.qcodes_adaptions.ZINode',
 'name': 'mfli1_sigouts0_amplitudes0'}
```
However, I could set the amplitude with `mfli1.sigouts[0].amplitudes[1].value(0.05)` 
```
> mfli1.sigouts[0].amplitudes[0].snapshot()
```
<img width="148" alt="image" src="https://user-images.githubusercontent.com/17175136/195816996-80f001b6-2775-4b6c-b07b-8df90bcbef1a.png">

Is it intentional that it's not zero-based index?

**Comment by tobiasah** [comment]
@einsmein Thanks for reaching out to us. This looks weird ... it should be zero-based indexed by all means 😇 

However, I could not reproduce it on my machine. I can both read and write to the `ampilitudes[0].value` node. 

The snapshot returns the follwoing: 

```json
{
    "functions": {},
    "submodules": {},
    "parameters": {
        "value": {
            "__class__": "zhinst.qcodes.qcodes_adaptions.ZIParameter",
            "full_name": "zi_baseinstrument_devxxxx_sigouts0_amplitudes0_value",
            "value": 0.100006103515625,
            "raw_value": 0.100006103515625,
            "ts": "2022-10-14 12:14:01",
            "unit": "V",
            "name": "value",
            "label": "value",
            "post_delay": 0,
            "instrument": "zhinst.qcodes.qcodes_adaptions.ZINode",
            "instrument_name": "zi_baseinstrument_devxxxx_sigouts0_amplitudes0",
            "inter_delay": 0
        }
    },
    "__class__": "zhinst.qcodes.qcodes_adaptions.ZINode",
    "name": "zi_baseinstrument_devxxx_sigouts0_amplitudes0"
}
```

May I ask for the following information: 

* `qcodes` version.
* `zhinst-qcodes` version.
* `zhinst-toolkit`version. 
* `zhinst-core` version (if you are using LabOne < 22.08 the version of `zhinst` instead)
* LabOne version
* Firmware revision

Just so that I can try to reproduce it.

**Comment by mehdialem** [comment]
@einsmein and @tobiasah, Thank you for the discussion. Let me clarify that both of you have correct observation but the difference is in the installed options on your instruments. @einsmein's device does not include the [MF-MD](https://www.zhinst.com/products/mf-md-multi-demodulator?from=MFLI) option while @tobiasah's device has it. The screenshots below shows the differences in the UI:

Device w/o MF-MD option
![mfli_base](https://user-images.githubusercontent.com/29656333/195830008-cbe20d99-dec6-468d-b9c0-6469ddaf6a30.PNG)

Device w/ MF-MD option
![mfli_md](https://user-images.githubusercontent.com/29656333/195830069-77520eec-d5a0-4ad2-ad8b-b54eb57c22a3.PNG)

 As evident from the screenshots, MFLI with MD option has 4 signal amplitudes indexed as `0`, `1`, `2`, and `3`. However, the device without MD option has only 1 signal amplitude which is the index `1` out of those 4 amplitudes. The reason is to decouple the phase of reference for demodulator (index `0`) from the phase of signal output (index `1`). 

So in summary, the user of MFLI without MF-MD option has only one choice for output amplitude and it is index `1`

**Comment by einsmein** [comment]
@tobiasah @mehdialem Thanks for the quick reply, and for the explanation. I still don't understand why the output signal params have to be at index 1 instead of 0, but I assume that comes from some lower-level implementation.

**Comment by mehdialem** [comment]
@einsmein, Let me explain it further using the screenshot below. One would like to have independent phases for signal generation (red-box phase in the screenshot with index `1`) and for signal demodulation (green-box phase in the screenshot with index `0`). Now suppose that the amplitude has an index of `0` in your script and you upgrade your MFLI with the MF-MD option. In this case your old script results in a different behavior for the new situation because it uses index `0` which means the signal output has the same phase of demodulator (green box). So this kind of indexing avoids behavioral incompatibility between instruments with and without options. I hope it answers your question. 

![mfli_base](https://user-images.githubusercontent.com/29656333/195863590-367a9c0f-9775-466d-997c-22fa00cc0697.PNG)
